bin/kubectl-crossplane-stack-init

```diff
@@ -170,27 +170,18 @@ function check_for_existing_stack {
 function create_build {
   cat > stack.Dockerfile <<'EOF'
 # Build the manager binary
-FROM golang:1.12.5 as builder
-
+FROM golang:1.13.6 as builder
 WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
+COPY . .
```

Comment:
Nitpick - this will also copy files such as the `.git` folder, which people may not want as it would increase the size of the image

Reply:
A [`.dockerignore`](https://docs.docker.com/engine/reference/builder/#dockerignore-file) file would remedy this.

Reply:
@displague thank you for the suggestion, I added a section to append `.git` to `.dockerignore`